addons/optionsmenu/config.cpp

```diff
@@ -32,3 +32,10 @@ class CfgAddons {
 class ACE_Extensions {
     extensions[] += {"ace_clipboard"};
 };
+
+
+class CfgCommands {
+    allowedHTMLLoadURIs[] = {
```

Comment:
This doesn't influence the one in description.ext at all.
+= could actually be used here though. Would be a good idea. That way other addons could use it too.